src/ProxyManager/Generator/Util/UniqueIdentifierGenerator.php

```diff
@@ -41,15 +41,8 @@
      */
     public static function getIdentifier(string $name) : string
     {
-        return str_replace(
-            '.',
-            '',
-            uniqid(
-                preg_match(static::VALID_IDENTIFIER_FORMAT, $name)
-                ? $name
-                : static::DEFAULT_IDENTIFIER,
-                true
-            )
-        );
+        $identifier = preg_match(static::VALID_IDENTIFIER_FORMAT, $name) ? $name : static::DEFAULT_IDENTIFIER;
+
+        return $identifier . substr(hash('sha256', $identifier), 0, 8);
```

Comment:
Note that no security constraints apply here, so I don't think we need this, still good improvement 👍

Reply:
I know but using MD5 anywhere will just set a bad precedent. It’s easier to explain "never use MD5 for anything" 😄